README.md

```diff
@@ -128,10 +137,11 @@ Follow these steps to run Airkeeper locally:
 ### Running Airkeeper on AWS Lambda
 
 Airkeeper is meant to be deployed to the AWS lambda service and for this you will need to add your credentials to the
-`config/aws.env` file. Once done, the `export-aws-env.sh` script can be used to load them into the environment.
+`config/aws.env` file. Once done, the [`export-aws-env.sh`](scripts/export-aws-env.sh) script can be used to load them
+into the environment.
```

Comment:
I'm a bit confused. After reading your comment I thought that you've removed the script altogether.

Reply:
Good catch. I forgot to remove this part after removing the script 👏🏻

Reply:
But the script itself is still there. At least according to GitHub :smile:

Reply:
LOL 🤦🏻 ... deleting file